raspinfo/raspinfo

```diff
@@ -109,3 +109,11 @@ echo "---------"
 echo
 
 dmesg | sed -e "s/\([0-9a-fA-F]\{1,4\}:\)\{7,7\}[0-9a-fA-F]\{1,4\}/y.y.y.y.y.y.y.y/g" | sed -e "s/[0-9a-fA-F]\{1,4\}:\(:[0-9a-fA-F]\{1,4\}\)\{1,4\}/y::y.y.y.y/g" | sed -e "s/\([0-9a-fA-F]\{2,2\}\:\)\{5,5\}[0-9a-fA-F]\{2,2\}/m.m.m.m/g"
+
+
+echo
+echo "eeprom (RPi 4)"
```

Comment:
Can we capitalise "EEPROM"?

Reply:
And add a few more dashes to underline it?

Reply:
Done